src/webgpu/util/f32_interval.ts

```diff
@@ -181,13 +175,17 @@ interface TernaryToIntervalOp {
   // All current ternary operations that are used in inheritance (clamp*) are
   // monotonic, so extrema calculation isn't needed. Re-using the Op interface
   // pattern for symmetry with the other operations
+
+  // All current ternary operations have a domain of the full f32 space, so a domain? override is not currently
```

Comment:
nit: Not sure this is technically an `override`. Suggest:

`// All current ternary operations have a domain of the full f32 space, so a domain is not currently provided.`